Makefile

```diff
@@ -1197,13 +1197,13 @@ mariadb_kuttl: input deploy_cleanup mariadb mariadb_deploy_prep ## runs kuttl te
 	make mariadb_cleanup
 
 .PHONY: kuttl_db_prep
-kuttl_db_prep: input deploy_cleanup mariadb mariadb_deploy mariadb_deploy_validate infra memcached_deploy ## installs common DB service(MariaDB and Memcached)
+kuttl_db_prep: input deploy_cleanup mariadb infra mariadb_deploy mariadb_deploy_validate memcached_deploy ## installs common DB service(MariaDB and Memcached)
 
 .PHONY: kuttl_db_cleanup
 kuttl_db_cleanup: memcached_deploy_cleanup infra_cleanup mariadb_deploy_cleanup mariadb_cleanup input_cleanup
 
 .PHONY: kuttl_common_prep
-kuttl_common_prep: kuttl_db_prep rabbitmq rabbitmq_deploy keystone keystone_deploy ## installs common middleware services and Keystone
+kuttl_common_prep: input deploy_cleanup mariadb infra rabbitmq keystone mariadb_deploy mariadb_deploy_validate memcached_deploy rabbitmq_deploy keystone_deploy ## installs common middleware services and Keystone
 
 .PHONY: kuttl_common_cleanup
 kuttl_common_cleanup: keystone_cleanup rabbitmq_cleanup kuttl_db_cleanup
```

Comment:
IMO, we should not list individual targets instead of using kuttl_db_cleanup.

Reply:
Agreed, I'll add a commit to this PR